plugins/plugin-chart-echarts/src/Pie/controlPanel.ts

```diff
@@ -119,7 +119,7 @@ const config: ControlPanelConfig = {
               min: 10,
               max: 100,
               step: 1,
-              default: 80,
+              default: 50,
```

Comment:
Bycatch: making the default pie slightly smaller, as labels outside will otherwise easily overflow the canvas if a chart is made very small.